src/main/kotlin/platform/AbstractModuleType.kt

```diff
@@ -36,6 +37,8 @@ abstract class AbstractModuleType<out T : AbstractModule>(val groupId: String, v
 
     abstract val id: String
 
+    abstract val supportedLanguages: List<LanguageType>
```

Comment:
I don't think this information should be in ModuleTypes, but rather in ProjectConfig. I don't see  any other use for this information other than for the project creator.